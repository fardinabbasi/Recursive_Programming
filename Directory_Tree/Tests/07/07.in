12 6
folder1 0
folder2 1
folder3 1
file1 1
file2 3
folder4 0
file3 3
file4 6
file5 0
folder5 2
file6 0
file7 10