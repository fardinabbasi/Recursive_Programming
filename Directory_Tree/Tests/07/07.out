|__folder1
|  |__folder2
|  |  |__folder5
|  |     |__file7
|  |__folder3
|  |  |__file2
|  |  |__file3
|  |__file1
|__folder4
|  |__file4
|__file5
|__file6